xmpsdk/include/TXMPUtils.hpp

```diff
@@ -769,7 +769,7 @@ template <class tStringObj> class TXMPUtils {
 									 XMP_StringPtr                separator,
 									 XMP_StringPtr                quotes,
 									 XMP_OptionBits               options,
-									 tStringObj *                 catedStr );
+									 tStringObj *                 concatenatedStr );
```

Comment:
I don't see any particular reason to shorten this token, but I'm happy to drop it (or any other change(s)) if requested.

Reply:
I would vote to drop all the changes done inside the **xmpsdk** folder. This is a 3rd party dependency in the project that was included in our codebase a long time ago, but we try to not touch these files too much.